ci-conda.Dockerfile

```
@@ -149,26 +153,11 @@ mv gh_*/bin/gh /usr/local/bin
 rm -rf gh_*
 EOF
 
-# Install codecov binary
+# Install codecov from source distribution
 ARG CODECOV_VER
 RUN <<EOF
-curl https://uploader.codecov.io/verification.gpg --max-time 10 --retry 5 | gpg --no-default-keyring --keyring trustedkeys.gpg --import
-
-case "${TARGETPLATFORM}" in
-  "linux/amd64") codecov_url="https://uploader.codecov.io/v${CODECOV_VER}/linux/codecov" ;;
-  "linux/arm64") codecov_url="https://uploader.codecov.io/v${CODECOV_VER}/aarch64/codecov" ;;
-  *) echo 'Unsupported platform' && exit 1 ;;
-esac
-
-curl -Os --max-time 10 --retry 5 ${codecov_url}
-curl -Os --max-time 10 --retry 5 ${codecov_url}.SHA256SUM
-curl -Os --max-time 10 --retry 5 ${codecov_url}.SHA256SUM.sig
-
-gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
-shasum -a 256 -c codecov.SHA256SUM
-chmod +x codecov
-mv codecov /usr/local/bin
-rm -f codecov.SHA256SUM codecov.SHA256SUM.sig
+# temporary workaround for discovered codecov binary install issue. See rapidsai/ci-imgs/issues/142
+pip install --no-binary codecov-cli codecov-cli==${CODECOV_VER}
```

Do we have to supply `--no-binary`? I think we would prefer to get the binary (wheel) on amd64 and let `pip` fall back to using the source distribution on arm64.
```suggestion
pip install codecov-cli==${CODECOV_VER}
```